pmd-visualforce/src/main/java/net/sourceforge/pmd/lang/vf/ast/VfTokenManager.java

```diff
@@ -23,10 +23,10 @@
 @Deprecated
 @InternalApi
```

Comment:
Not sure, whether we decided yet: Are the `*TokenManager` classes internal api or not? The JavaTokenManager isn't...

Reply:
I'll remove `@Deprecated` and keep `@InternalApi` for now...